comet/utils/debrid.py

```
@@ -168,6 +221,24 @@ async def get_cached_availability(
             AND ((CAST(:episode as INTEGER) IS NULL AND episode IS NULL) OR episode = CAST(:episode as INTEGER))
         """
         )
-        results = await database.fetch_all(query, params)
+        try:
+            results = await database.fetch_all(query, params)
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)